projects/angular-auth-oidc-client/src/lib/utils/url/url.service.ts

```diff
@@ -280,7 +278,7 @@ export class UrlService {
             return this.createAuthorizeUrl(codeChallenge, silentRenewUrl, nonce, state, 'none');
         }
 
-        this.loggerService.logWarning('authWellKnownEndpoints is undefined');
+        // this.loggerService.logWarning('authWellKnownEndpoints is undefined');

```

Comment:
Is this comment on purpose? Maybe a leftover? Can be put back in or removed completely?

Reply:
Yes, it is a leftover. I'm changing it now. Thanks for the heads up :)

Reply:
Yes, it is a leftover. I'm changing it now. Thanks for the heads up :)

```suggestion
        this.loggerService.logWarning('authWellKnownEndpoints is undefined');
```

Reply:
You're welcome. That is why we do reviews. but thank YOU for the PR! Highly appreciated!